content/src/content/jcr_root/apps/core/wcm/components/commons/editor/dialog/inherited/v1/inherited/render.jsp

```diff
@@ -0,0 +1,17 @@
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
why do we have render.jsp? I'd remove it.

Reply:
the render.jsp is directly called by the /libs/granite/ui/components/foundation/form/field component and is therefore required.